build.sbt

```diff
@@ -1,4 +1,4 @@
-import scala.sys.process._
+import scala.sys.process.*
```

Comment:
latest SBT itself uses scala 3 so I noticed warnings about the old syntax.

Reply:
No, sbt 1.x does _not_ use Scala 3, it still uses Scala 2.12.x. Latest sbt 1.9.7 upgraded to latest Scala 2.12.17. To help upgrading to Scala 3, once sbt switches to it (I guess that will be sbt 2.x), the recent Scala 2.12.x releases can now handle some Scala 3 syntax. This one here e.g. is available since [Scala 2.12.15](https://github.com/scala/scala/releases/tag/v2.12.15):

> and facilitates cross-building with Scala 3 with:
> ...
>- Allow `import x.{*, given}` under `-Xsource:3` (https://github.com/scala/scala/pull/9736 by [@smarter](https://github.com/smarter))

More Scala 3 compatibily was introduced in sbt [2.12.16](https://github.com/scala/scala/releases/tag/v2.12.16) and [2.12.17](https://github.com/scala/scala/releases/tag/v2.12.17) as well.

Reply:
that's super helpful because I was having a play upgrading this project to scala 3 after this (and gave up for now) and I was a bit confused when I was trying to work out what scalafix SBT plugin I needed